src/test/java/fi/metatavu/kuntaapi/test/server/integration/ptv/TestPtvConsts.java

```diff
@@ -2,76 +2,351 @@
+    "7fdd7f84-e52a-4c17-a59a-d7c2a3095ed5",
+    "3d1759a2-e47a-4947-9a31-cab1c1e2512b",
+    "b458d033-042d-4cbe-b030-34e81da28821"
   };
 
-  public static final String[] SERVICES = {
-    "2f21448e-e461-4ad0-a87a-47bcb08e578e",
-    "bcfaf5f1-98db-401e-8628-7d4f2b4ec260",
-    "4c490d84-1fc2-4011-98b5-fbc7a878cbce",
-    "b21dbc0e-ea54-449d-8419-3fe7a6126948",
-    "6b979930-048e-4ed4-a80c-0e11ba562dfe",
-    "58a33fee-d7c6-4875-a1b7-75d70b00d012",
-    "08ab5b97-d04d-4d16-a3f3-dac654b140ec",
-    "2fb268a5-8fa5-467e-b908-98131b916365",
-    "8f0fe030-794e-4ed5-beb7-57c6be43f2ba"
+  public final static String[] SERVICES = {
+    "9f242076-de0b-4c58-810e-aac4979d71fa",
+    "7be2de65-f022-4985-84ef-cabf9df574ab",
+    "9e63c517-812a-42a0-9e5b-3c1a0f0ff630",
+    "47e7c82b-9d67-4a7e-93f7-c6ff87adf2fa",
+    "7cf9a776-808e-4e58-a904-863f48887ea2",
+    "c4f1632e-66a8-410b-94f0-9196e028c644",
+    "5fe23177-860c-4257-ac03-b4cdeab87d19",
+    "4b9f3ded-173b-4018-b43b-d3bbd1ef0ba8",
+    "a683d5eb-ee1a-4869-98ba-a4a00ed880da",
+    "f8d7f555-a02c-47b3-9219-7f36767b979a",
+    "d088280f-3d17-4698-b440-feffc6e4fa0f",
+    "f4e22166-0b6a-4d33-b885-715a298999c4",
+    "2ba22c86-e28a-4988-aca7-38c6ce95bed7",
+    "228738ee-754a-49b7-8240-2b84c5ca2970",
+    "0a94d54d-febd-4149-bceb-f3cc0fd648e6",
+    "17c633dd-5408-495a-86c1-d8f57598e899",
+    "b7fb6633-d754-45fb-8241-35f22b75c2dd",
+    "e45cae1e-dfbc-44eb-b57b-ef73f21b38c4",
+    "1ba9c665-d20d-4ea0-8282-81e35c701fe7",
+    "73ceb656-c1a3-4479-b84a-d99d88f1b66d",
+    "8062afa8-41c4-44c0-b1f7-d86c0891e9e3",
+    "610325e0-f485-4caa-bd3b-5bf74d5480c5",
+    "83e5fa7f-a22c-43b7-aa27-06474d21ec59",
+    "6ccb671f-05cd-46cf-a0a4-4f98d43dfd37",
+    "f20366fd-b41d-406b-bfab-e6928359a008",
+    "faa6b1ed-65e0-415e-9db4-67021d8eebc0",
+    "ea3af438-bc77-49da-8f0c-2f217eee491f",
+    "bedd6808-8472-4f82-a412-e9521f14c621",
+    "0739289c-cc26-4097-b029-fa9c58c8d4e9",
+    "d2ea56fe-1bca-4d26-8579-875a4c6c9638"
   };
 
-  public static final String[] SERVICE_CHANNELS = {
-    "b8e980f4-58c7-477b-a9d1-b01d50be50f0",
-    "0452df6a-f322-47de-af5b-767694f120cd",
-    "faa9c7b7-5b1f-4964-b04f-410558b66647",
-    "047b91ef-aa08-4be6-b72d-1231e61c5cb9",
-    "97c01787-b57c-42f1-a002-7d09b363993f",
-    "74b3ac81-cf57-40b0-b0b1-b9dcf752a143",
-    "b5fba182-2a08-44aa-a168-c11f8d1d4664",
-    "614bdc8a-bb55-4f36-be4e-7e1bea9a7706",
-    "dda27a67-1b20-4966-a57e-162ca4c54ef5",
-    "4d52960e-2e39-4d9e-8f7c-5bc8994b024f",
-    "0c00262c-3529-4cc3-b0fe-2f03a2c4e4ce",
-    "1dec7c89-96b8-4860-83d0-bad8b8897ba6",
-    "a31fbcc7-d45f-4eb2-a32a-afd134d02aa5",
-    "b0ca2d94-f113-47ae-9c90-1bc1733c44be",
-    "1af0ec8c-b647-4f40-b1a9-db6052c5096d",
-    "155b896e-f1f5-43db-9497-aa3a8afd2487",
-    "0e4abaca-f57e-4649-a42d-0f421d09e9fd"
+  public final static String[] SERVICE_CHANNELS = {
+    "1bd39cda-3006-482e-acff-93ab8ddc9f1b",
+    "3b85838e-7ee7-4123-ba50-3c901e0d1597",
+    "bc91db9e-2f56-4044-a375-30987de714eb",
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "bc91db9e-2f56-4044-a375-30987de714eb" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)